imap_processing/cdf/utils.py

```diff
@@ -121,6 +128,13 @@ def write_cdf(dataset: xr.Dataset, **extra_cdf_kwargs: dict) -> Path:
     dataset.attrs["Logical_file_id"] = file_path.stem
     # Add the processing version to the dataset attributes
     dataset.attrs["ground_software_version"] = imap_processing._version.__version__
+    # Add any parent files to the dataset attributes
+    if parent_files:
+        # Include the current files if there are any and include just the filename
+        # [file1.txt, file2.cdf, ...]
+        dataset.attrs["Parents"] = dataset.attrs.get("Parents", []) + [
+            p.name for p in parent_files
```

Comment:
```suggestion
            parent_file.name for parent_file in parent_files
```

My preference here is to be more explicit.  But you can take it or leave it!